pytorch_lightning/trainer/trainer.py

```diff
@@ -589,6 +590,7 @@ def predict(
         dataloaders: Optional[Union[DataLoader, List[DataLoader]]] = None,
         datamodule: Optional[LightningDataModule] = None,
         return_predictions: Optional[bool] = None,
+        ckpt_path: Optional[str] = 'best',
```

Comment:
this sounds a bit confusing, what if someone called the model best too?

Reply:
Valid point, but out of scope for this PR